### ISSUE TITLE
[Backoffice] Amélioration de l'affichage

### ISSUE THREAD

**Issue by mvergez** [PnX-SI/GeoNature-citizen#329]
Nous proposons de modifier l'affichage du backoffice :  

Avant : 
![image](https://user-images.githubusercontent.com/85738261/146761448-e77b7d04-659b-4f8f-ae68-a73ef23cc5fa.png)

Après : 
![image](https://user-images.githubusercontent.com/85738261/146761178-2b7c6846-6f78-406e-b91c-a095a99b12f6.png)

Cet affichage peut être réglé par la variable d'environnement : `FLASK_ADMIN_FLUID_LAYOUT` à mettre à `true` dans `default_config.toml`
Je fais une PR dans ce sens.

**Comment by lpofredc** [comment]
Oui, c'est une bonne chose, c'était dans ma todolist. Je ne mettrai pas d'option, on le met en `fluid` et c'est tout ;)

**Comment by mvergez** [comment]
Super, merci pour ton retour.

Je mets FLASK_ADMIN_FLUID_LAYOUT = true dans le config_toml pour qu'il soit mis automatiquement dans la config de Flask non ?
Tu pensais la renseigner où ?
J'envoie la PR

**Comment by lpofredc** [comment]
Maintenant en paramètre par défaut, sans config spécifique.
